not-to-release/sources/weblog/blogspot.com_rigorousintuition_20060511134300_ENG_20060511_134300.xml.conllu

```diff
@@ -3370,7 +3370,7 @@
 5	truth	truth	NOUN	NN	Number=Sing	3	nmod	3:nmod:of	SpaceAfter=No
 6	,	,	PUNCT	,	_	3	punct	3:punct	_
 7	ala	ala	ADP	IN	_	8	case	8:case	_
-8	satanism	satanism	PROPN	NNP	Number=Sing	3	nmod	3:nmod:ala	SpaceAfter=No
+8	satanism	satanism	NOUN	NN	Number=Sing	3	nmod	3:nmod:ala	SpaceAfter=No
```

Comment:
as the name of a religion I would call it a proper name (by analogy to Judaism etc.)

Reply:
Again not sure for satanism (and other religions). If we tag these as PROPN why "communism" or "neo-liberalism" shouldn't be tagged as PROPN as well?

Reply:
Names of religions (but not necessarily political ideologies) are conventionally capitalized and treated as proper nouns in English. If the PROPN guidelines don't address this specifically I think the English corpora should at least try to be consistent with one another and default to precedents such as PTB.

Reply:
OK, convinced, I was not aware of this. I'll reset this to PROPN

Reply:
BTW, also treated as proper nouns in English are months and days of the week. I know these are not capitalized in French. It would be a major undertaking to develop detailed and crosslinguistically uniform policies for NOUN vs. PROPN, but for now we should default to language-specific conventions. :)

Reply:
Yes, I was I aware of that, that's why I didn't touch them :-). Indeed would be nice to have some universal policies, but I agree to stick for the time being to language-specific conventions. I saw the discussions on film titles and how to tag them. Even within a single language it's not always obvious.

Reply:
+1 for staying as close as possible to PTB guidelines (better to have a questionable but consistent guideline than every English corpus being slightly different from every other one...)